### ISSUE TITLE
Need for a feedback on pressing the Bookmark button

### ISSUE THREAD

**Issue by llemeurfr** [readium/readium-desktop#571]
No screen reader feedback on pressing the Bookmark button. 
A keystroke should be provided to insert the bookmark so that the user can place the bookmark without losing the reading position.

TODO: study the keyboard shortcut which could trigger that info.

**Comment by llemeurfr** [comment]
source comment = Daisy, anno-010 Add a Bookmark or Highlight.
